app/views/messages/_message.html.erb

```diff
@@ -45,123 +49,177 @@
         <%= message.user? ? 'You' : message.assistant.name %>
       </div>
 
+      <turbo-frame id="message-<%= message.id %>">
+
       <div class="hidden" data-clipboard-target="text"><%= message.content_text %></div>
```

Comment:
All the diff the remaining diff in this file is simply indenting the block since it's now inside the turbo-frame